app/javascript/packs/dashboard_new_accounts_widget.js

```diff
@@ -0,0 +1,17 @@
+import { NewAccountsWidgetWrapper } from 'Dashboard/components/NewAccountsWidget'
+import { safeFromJsonString } from 'utilities'
+
+const containerId = 'new-accounts-widget'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const container = document.getElementById(containerId)
```

Comment:
I recommend asserting the container exists like it's done in other packs. Flow will warn you anyway. 👮🏻‍♂️

Reply:
Cool! And how do we do that? Is there an example in code I can check out?

Reply:
Actually, let's favor this other one, better for debugging https://github.com/3scale/porta/blob/01bf13b1a866bde19075b2f3c341def9a7c6427d/app/javascript/packs/backend_api_metrics_index.js#L13-L15

Reply:
Nice stuff!
Added in 73c8a61.